### PR TITLE
feat(theme): home features section layout consistent

### PR DESCRIPTION
**Pull request by zonemeen**
close https://github.com/vuejs/vitepress/issues/2318

### CODE REVIEW

File: src/client/theme-default/components/VPFeatures.vue [hunk 1]
```diff
@@ -26,7 +26,7 @@ const grid = computed(() => {
     return 'grid-3'
   } else if (length % 3 === 0) {
     return 'grid-6'
-  } else if (length % 2 === 0) {
+  } else if (length % 2 === 0 || length % 2 === 1) {
```
Comment (brc-dd) [inline src/client/theme-default/components/VPFeatures.vue:29]: Just a simple else should work here, right? (as `length % 2 === 0 || length % 2 === 1` will always be true)

```suggestion
  } else {
```

**Comment by zonemeen** [comment]
@brc-dd  And when the length is one, should it be set to `grid-1`? Like this:

image

Otherwise it is like this:

image

I think it is more reasonable to set the style as `grid-1` when the length is one.

**Comment by brc-dd** [comment]
Yeah for length = 1, `grid-1` seems fine.

**Comment by zonemeen** [comment]
Yeah, I have resubmitted the code.😀

**Review by brc-dd — APPROVED** [review]

**Comment by brc-dd** [comment]
Removed the grid-1 thing, because there is no CSS corresponding to that -- it is the default behavior. We just need to make sure to not apply grid-4 on that case. Updated the condition accordingly.

**Pull request merged**